utils/docker-tests/Dockerfile

```diff
@@ -3,13 +3,12 @@ FROM centos:7
 VOLUME /payload
 
 RUN yum update -y && \
-    yum install python-virtualenv make git -y
+    yum install python-virtualenv python-setuptools make git -y
 
 # NOTE(ivasilev) Unless we upgrade pip an obsolete v 9 will be on the system
 # and we need at least 10.0.1 to install a not-yet-merged framework pr in case
 # it's necessary
-RUN curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py" && \
-    python get-pip.py
+RUN easy_install pip
```

Comment:
what is the reason for this change?

Reply:
@Rezney  https://github.com/oamg/leapp-repository/pull/637 - rebase will drop this commit as it's already present in master.